<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajout_employes</class>
 <widget class="QMainWindow" name="ajout_employes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-70</y>
      <width>761</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/icons/Inter.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>384</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(27, 75, 104);
color:white;
border-radius:15px;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prenom">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mdp">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>323</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>role</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>334</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mdp</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_role">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);</string>
    </property>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Technicien</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mecanicien</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Employe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Secretere</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>49</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ajout employees</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 154, 168);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_cin</tabstop>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>comboBox_role</tabstop>
  <tabstop>lineEdit_mdp</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
